src/pages/home/report/ReportActionsList.js

```diff
@@ -295,12 +311,26 @@ function ReportActionsList({
 
     const trackVerticalScrolling = (event) => {
         scrollingVerticalOffset.current = event.nativeEvent.contentOffset.y;
-        handleUnreadFloatingButton();
         onScroll(event);
+
+        if (isNative) {
```

Comment:
We only call the `handleUnreadFloatingButton` for Native devices because the [native InvertedFlatList](https://github.com/wlegolas/ExpensifyApp/blob/main/src/components/InvertedFlatList/index.native.js) doesn't have the implementation to call the `onScrollEnd` handler.

Reply:
This check is only used to have the same behavior (shows the "New message" button on scroll) for Native devices because there is no scroll implementation for the `index.native.js` file for InvetedFlatList.

I'll improve the `index.native.js` to use the scrolls mechanism instead of this `isNative` check.